.env.ex

```diff
@@ -1,2 +0,0 @@
-# .env
-DISCORD_TOKEN = <your-bot-token>

```

Comment:
Actually, don't delete this file. Let it be as an example for future contributors.

Reply:
Mistake on my part, when the dev and main merged on my fork I think it deleted the file.